src/numberArray.js

```diff
@@ -0,0 +1,14 @@
+export default function(a, b) {
+  if (!b) b = [];
+  var n = a ? Math.min(b.length, a.length) : 0,
+      c = b.slice(),
+      i;
+  return function(t) {
+    for (i = 0; i < n; ++i) c[i] = a[i] * (1 - t) + b[i] * t;
+    return c;
+  };
+}
+
+export function isNumberArray(x) {
+  return ArrayBuffer.isView(x) && !(x instanceof DataView);
```

Comment:
Hi!  Just wanted to flag that `typeof ArrayBuffer.isView === 'undefined'` in PhantomJS.    I know it's a super old browser and I'm not really expecting D3 to be perfectly compatible with it, but just commenting here in case someone else gets the same error.   Thanks!

re: https://github.com/openstreetmap/iD/issues/7072#issuecomment-559147596

Reply:
we could test for it?

`return ArrayBuffer.isView && ArrayBuffer.isView(x) && !(x instanceof DataView);`